Controls/Input/Input.UWP/AutoCompleteBox/AutoCompleteTextBox.cs

```diff
@@ -17,6 +25,36 @@ public AutoCompleteTextBox()
             this.DefaultStyleKey = typeof(AutoCompleteTextBox);
         }
 
+        internal void UpdateDeleteButtonVisibility()
+        {
+            if (string.IsNullOrEmpty(this.Text))
+            {
+                this.deleteButton.Visibility = Visibility.Collapsed;
+                return;
+            }
+
+            this.deleteButton.Visibility = this.isClearButtonVisible == true ? Visibility.Visible : Visibility.Collapsed;
```

Comment:
We do not need that this.isClearButtonVisible == true. The isClearButtonVisible field is not bool? and because of that we could simply do this.isClearButtonVisible ? Visibility.Visible : Visibility.Collapsed